src/main/java/com/sap/sgs/phosphor/fosstars/data/github/ScansForVulnerableDependencies.java

```diff
@@ -1,26 +1,28 @@
 package com.sap.sgs.phosphor.fosstars.data.github;
 
 import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.SCANS_FOR_VULNERABLE_DEPENDENCIES;
+import static java.lang.Boolean.FALSE;
 
 import com.sap.sgs.phosphor.fosstars.model.Feature;
 import com.sap.sgs.phosphor.fosstars.model.Value;
+import com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures;
 import com.sap.sgs.phosphor.fosstars.tool.github.GitHubProject;
 import java.io.IOException;
 import java.util.Arrays;
 import java.util.List;
 
 /**
- * This data provider tries to fill out the
- * {@link
- * com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures#SCANS_FOR_VULNERABLE_DEPENDENCIES}.
- * feature.
- * It is based on a number of data providers:
+ * This data provider tries to fill out the {@link OssFeatures#SCANS_FOR_VULNERABLE_DEPENDENCIES}.
+ * feature. It is based on the following data providers:
```

Comment:
data provider?

Reply:
The list currently contains one data provider but it's still a list.